slrealizer/utils/utils.py

```diff
@@ -1,23 +1,58 @@
-#==============================================
-
+#import numpy as np
+#from fractions import Fraction
+#import math
 import numpy as np
-import pandas
-import desc.slrealizer
-from fractions import Fraction
-import math
+#import pandas as pd
+
+def e1e2_to_ephi(e1, e2):
+    e = np.power(np.power(e1, 2.0) + np.power(e2, 2.0), 0.5)
+    phi = np.arctan(e2/e1)
```

Comment:
Should be `phi = 0.5 * np.arctan ...` to include the factor of two that makes ellipticity a spinor, not a vector

Reply:
Done